automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSpec.scala

```diff
@@ -38,6 +38,7 @@ class LeonardoSpec extends FreeSpec with Matchers with Eventually with ParallelT
   val project = GoogleProject(LeonardoConfig.Projects.default)
   val sa = GoogleServiceAccount(LeonardoConfig.Leonardo.notebooksServiceAccountEmail)
   val bucket = GcsBucketName("mah-bukkit")
+  val incorrectJupyterExtensionUri = "gs://leonardo-swat-test-bucket-do-not-delete"
```

Comment:
This is just a bucket. Doesn't it need to be a nonexistent file inside this bucket?

Reply:
Passing an empty bucket is causes cluster creation to error, I haven't tried it with a non existent file. Is there a reason to use a non existent file instead?

Reply:
A file and a bucket aren't the same thing and the error is almost certainly different. Also, isn't that the bucket we made yesterday? I thought we made that public? Or did you set it back to private?